### PR TITLE
Fixed Request Factory Example

### PR DESCRIPTION
**Pull request by vdambeck**
The Request Factory Example dosen't work anymore.
I change the example code. So it will work now.

### CODE REVIEW

**Review by roman-1983 — APPROVED** [review]
👍

**Review by HeahDude — APPROVED** [review]
Nice, thanks!

File: components/http_foundation.rst [hunk 1]
```diff
@@ -303,7 +304,7 @@ PHP callable that is able to create an instance of your ``Request`` class::
         array $server = array(),
         $content = null
     ) {
-        return SpecialRequest::create(
+        return new SpecialRequest(
```
Comment (HeahDude) [inline components/http_foundation.rst]: Why not keeping the static method?

File: components/http_foundation.rst [hunk 2]
```diff
@@ -293,6 +293,7 @@ methods or changing some default behavior might help. In that case, register a
 PHP callable that is able to create an instance of your ``Request`` class::
 
     use Symfony\Component\HttpFoundation\Request;
+    use AppBundle\Classes\SpecialRequest;
```
Comment (HeahDude) [inline components/http_foundation.rst]: Maybe `Http` would be more appropriate as namespace than `Classes?`

**Comment by xabbuh** [comment]
Thank you @vdambeck.

**Pull request closed without merging**